src/scripts/admin/admin.store.js

```diff
@@ -90,6 +91,45 @@ let UserStore = Reflux.createStore({
     },
 
     /**
+     * Search username and email
+     *
+     */
+    searchUsername(searchInput){
+        let users = this.data.users;
+
+        for (let user of users) {
+            user.inList = true;
+            if(user.email.toLowerCase().includes(searchInput.toLowerCase()) || user.lastname.toLowerCase().includes(searchInput.toLowerCase()) || user.firstname.toLowerCase().includes(searchInput.toLowerCase())){
```

Comment:
Clean up this if condition so it's easer to read. A couple returns can clean it up like this

	if (
		user.email.toLowerCase().includes(searchInput.toLowerCase())    ||
		user.lastname.toLowerCase().includes(searchInput.toLowerCase()) ||
		user.firstname.toLowerCase().includes(searchInput.toLowerCase())
	) {}

and then you can use variables to reduce the redundancy a little

	searchInput = searchInput.toLowerCase();

	if (
		user.email.toLowerCase().includes(searchInput)    ||
		user.lastname.toLowerCase().includes(searchInput) ||
		user.firstname.toLowerCase().includes(searchInput)
	){}